test/units/testsuite-70.sh

```diff
@@ -29,6 +29,14 @@ env PIN=123456 /usr/lib/systemd/systemd-cryptsetup attach test-volume $img - tpm
 # Check failure with wrong PIN
 env PIN=123457 /usr/lib/systemd/systemd-cryptsetup attach test-volume $img - tpm2-device=auto,headless=1 && { echo 'unexpected success'; exit 1; }
 
+# Check LUKS2 token plugin unlock (i.e. without specifying tpm2-device=auto)
```

Comment:
Isn't there a need for unexpected success test for the external token as well?

Reply:
Thank you for your review! I would argue that most of the code paths are already covered by the other tests because if LUKS2 token plugin support is available, `attach_luks_or_plain_or_bitlk_by_tpm2` calls `attach_luks2_by_tpm2_via_plugin`, which in turn uses `crypt_activate_by_token_pin` (i.e. the same function that is called when directly using the LUKS2 token plugin, like this test does). On the other hand, a quick check whether using a wrong PIN leads to a failure doesn't really hurt, so I have implemented in an updated version of the PR :)